crates/ruff_python_semantic/src/all.rs

```diff
@@ -80,7 +80,7 @@ where
         flags: &mut DunderAllFlags,
     ) {
         for elt in elts {
-            if let Expr::StringLiteral(ast::ExprStringLiteral { value, range }) = elt {
+            if let Expr::StringLiteral(ExprStringLiteral { value, range }) = elt {
```

Comment:
micro-nit: I personally prefer referring to all AST nodes by their fully qualified names outside the `ruff_python_ast` crate, as it means that you don't have to continually update the `use ruff_python_ast::{x, y, z, ...}` line at the top when you discover in a future refactor that you need access to more nodes.